### PR TITLE
Feat psdata kriti

### PR DESCRIPTION
**Pull request by Suryakandukuri**

### CODE REVIEW

**Review by Suryakandukuri — COMMENTED** [review]
Please add documentation for all the states as to where the data is taken from and everything else.

File: projects/psdata/src/ConsolidationScriptMizoram.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        path_parts = file_path.split("/")
+        for i in range(1, len(path_parts) + 1):
+            present_path = "/".join(path_parts[:i])
+            Path(present_path).mkdir(exist_ok=True)
+
+    # logic to consolidate all the files should come here
+
+
+@click.command()
+@click.option("-state", default="1_WestBengal", help="State_Name")
```
[the remark this answers is not in the document]
Comment (kritimaloo) [reply projects/psdata/src/ConsolidationScriptMizoram.py:57]: Sir, West Bengal is just the default state name. The state input was given as Mizoram while running the script.

File: projects/psdata/src/data/psdata_reports.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+5. [Meghalaya](http://ceomeghalaya.nic.in/erolls/electoral-rolls-pdf.html)
+
+## **Scripts-**
+(**Path-** [bipp-datasets/projects/psdata/src/data/](https://github.com/bippisb/bipp-datasets/tree/feat_psdata_kriti/projects/psdata/src/data))
+1. psdataStateName.py scripts in the src/data folder are used to scrape polling station data for each particular state.
+2. Scripts for consolidating assembly constituency files for each state as well as on combined file are also present under src/data folder.
+
+## **To Run Scrpits-**
+1. Install Python
+2. Install required packages
```
Comment (Suryakandukuri) [inline projects/psdata/src/data/psdata_reports.md:22]: Please put poetry steps in here

**Review by Suryakandukuri — COMMENTED** [review]
Please add the poetry steps as mentioned in the comment

**Pull request merged**